client/components/mma/shared/ProductInfoTableV2.tsx

```diff
@@ -0,0 +1,83 @@
+import { css } from '@emotion/react';
+import {
+	from,
+	palette,
+	space,
+	textSans,
+	until,
+} from '@guardian/source-foundations';
+import { parseDate } from '@/shared/dates';
+import type { ProductDetail } from '@/shared/productResponse';
+
+export interface ProductInfoTableProps {
+	productDetail: ProductDetail;
+}
+
+export const ProductInfoTableV2 = (props: ProductInfoTableProps) => {
+	const valueBoxCss = () => {
+		return css`
+			background-color: ${palette.neutral[97]};
+			display: flex;
+			width: 803px;
+			${until.tablet} {
+				width: 351px;
+			}
+			${until.tablet} {
+			}
+			flex-wrap: wrap;
+			padding: ${space[4]}px ${space[6]}px;
+		`;
+	};
+
+	const tableEntryCss = () => {
+		return css`
+			${textSans.medium()};
+			margin-right: ${space[2]}px;
+		`;
+	};
+
+	const tableValueCss = () => {
+		return css`
+			color: #606060;
+			padding-right: 32px;
+			width: 100%;
+			${from.tablet} {
+				width: auto;
```

Comment:
just trying to understand, what do the different widths do at each screen size?

Reply:
This isn't required so I've removed it